src/main/java/com/github/yusufyilmazfr/podcast4j/factory/Podcast4jServiceFactory.java

```diff
@@ -35,13 +36,17 @@ public class Podcast4jServiceFactory {
     private static Podcast4jAppleReplacementService appleReplacementService;
     private static Podcast4jSearchService searchService;
 
-    private static final ObjectMapper objectMapper = new ObjectMapper();
-
     private Podcast4jServiceFactory() {
-        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
     }
 
     public static Podcast4jServiceFactory with(Config config) {
+        ObjectMapper objectMapper = null;
+        if (config.getObjectMapper() != null) {
+            objectMapper = config.getObjectMapper();
```

Comment:
`ObjectMapper ` in config, that is good!

Reply:
After calling the api extensively, I found two major deserialization anomalies. One is that when the object is empty, the PodcastIndexApi returns an empty array "[]" instead of "{}". The other is that the field that should be the url does not start with http or https, causing the parsing to the URL object to fail.
Therefore, I think it would be more flexible to customize the ObjectMapper.